api/v1alpha1/patch/datadogagent.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package patch
+
+import "github.com/DataDog/datadog-operator/api/v1alpha1"
+
+// DatadogAgent used to patch the current DatadogAgent instance to use the rescent fields.
```

Comment:
i think `rescent` is a typo?

Reply:
I I wanted to say `recent`

Reply:
that was my guess, but i also wasn't sure what you meant by "recent fields"

Reply:
recent == not deprecated (in my mind)